Makefile

```diff
@@ -3,13 +3,20 @@ GO_PACKAGES	=	$(foreach dir,$(GO_DIRS),./$(dir)/...)
 
 .PHONY: lint test vet setup
 
+.env-checked: bin/check-env
```

Comment:
Do me a favor and put this include and the rest of the block, at the bottom of the Makefile.  If we ever add anything to `.env-checked` it means it will have access to the rest of the Makefile's variables.